unittests/firmware/coins.cpp

```diff
@@ -173,7 +173,7 @@ TEST(Coins, BIP32AccountName) {
                  {0x80000000 | 44, 0x80000000 | 931, 0x80000000 | 69, 0, 0},
                  5,
                  true,
-                 "THORChain Account #69\nAddress #0"}};
+                 "THORChain Account #69"}};
```

Comment:
The `Address #n` was originally specified in a0250ff, and I don't think it was the originally intended behavior.